editor/src/components/canvas/canvas-strategies/strategies/convert-to-absolute-and-move-strategy.tsx

```diff
@@ -248,13 +255,18 @@ const getAutoLayoutSiblings = memoize(getAutoLayoutSiblingsInner, { maxSize: 1 }
 
 function getAutoLayoutSiblingsInner(
   jsxMetadata: ElementInstanceMetadataMap,
+  pathTrees: ElementPathTrees,
   targets: Array<ElementPath>,
 ): Array<ElementInstanceMetadata> {
   if (!targets.every((t) => EP.isSiblingOf(targets[0], t))) {
     // this function only makes sense if the targets are siblings
     return []
   }
-  return MetadataUtils.getSiblingsParticipatingInAutolayoutUnordered(jsxMetadata, targets[0])
+  return MetadataUtils.getSiblingsParticipatingInAutolayoutUnordered(
```

Comment:
Likewise with this, we already have an `Ordered` version of this function

Reply:
Fixed in d1565dc9d.